app/options.cpp

```diff
@@ -487,6 +501,8 @@ void Options::readSettings()
     m_spellingCheckEnabled = settings.value(SPELLINGCHECK_ENABLED, true).toBool();
     m_dictionaryLanguage = settings.value(DICTIONARY_LANGUAGE, "en_US").toString();
 
+    //Source viewer settings
```

Comment:
That was just a comment I left while trying to find the right spot to change the code. It shouldn't have been part of the commit at all :-( I removed it.